app/lib/features/tasks/widgets/task_list_item_card.dart

```diff
@@ -58,6 +63,7 @@ class TaskListItemCard extends ConsumerWidget {
 
   Widget expandable(BuildContext context, WidgetRef ref, TaskList taskList) =>
       ExpansionTile(
+        trailing: showOnlyTaskList ? const SizedBox() : null,
```

Comment:
I don't understand it. How is null supposed to be different from empty widget here?

Reply:
`ExpansionTile` shows arrows on trailing as default if value is null else it will show given widget.